src/processor/reconnection-graph.service.ts

```diff
@@ -89,60 +94,40 @@ export class ReconnectionGraphService {
       const exportedUpdates = graphState.exportUserGraphUpdates(dsnpUserId.toString());
 
       const providerKeys = createKeys(this.configService.getProviderAccountSeedPhrase());
-
-      let batch: SubmittableExtrinsic<'rxjs', ISubmittableResult>[] = [];
-      let batchItemCount = 0;
-      let batchCount = 0;
-      const batches: SubmittableExtrinsic<'rxjs', ISubmittableResult>[][] = [];
-      exportedUpdates.forEach((bundle) => {
+      const blockDelay = BlockchainConstants.SECONDS_PER_BLOCK * MILLISECONDS_PER_SECOND;
+      let statefulStorageTxHash: Hash;
+      let txMonitorJob: ITxMonitorJob;
+      let tx: SubmittableExtrinsic<'rxjs', ISubmittableResult>;
+      // eslint-disable-next-line no-restricted-syntax
+      for (const bundle of exportedUpdates) {
         switch (bundle.type) {
           case 'PersistPage':
-            batch.push(
-              this.blockchainService.createExtrinsicCall(
-                { pallet: 'statefulStorage', extrinsic: 'upsertPage' },
-                bundle.ownerDsnpUserId,
-                bundle.schemaId,
-                bundle.pageId,
-                bundle.prevHash,
-                Array.from(Array.prototype.slice.call(bundle.payload)),
-              ),
+            tx = this.blockchainService.createExtrinsicCall(
+              { pallet: 'statefulStorage', extrinsic: 'upsertPage' },
+              bundle.ownerDsnpUserId,
+              bundle.schemaId,
+              bundle.pageId,
+              bundle.prevHash,
+              Array.from(Array.prototype.slice.call(bundle.payload)),
             );
-            batchItemCount += 1;
-            // If the batch size exceeds the capacityBatchLimit, send the batch to the chain
-            if (batchItemCount === this.capacityBatchLimit) {
-              // Reset the batch and count for the next batch
-              batches.push(batch);
-              batchCount += 1;
-              batch = [];
-              batchItemCount = 0;
-            }
+            // eslint-disable-next-line no-await-in-loop
+            statefulStorageTxHash = await this.processSingleBatch(providerKeys, tx);
+            txMonitorJob = {
+              id: jobId,
+              txHash: statefulStorageTxHash,
+              epoch: currentCapacityEpoch.toString(),
+              lastFinalizedBlockHash,
+            };
+            this.logger.debug(`Adding job to graph change notify queue: ${txMonitorJob.id}`);
+            this.graphTxMonitorQueue.add(`Graph Change Notify Job - ${txMonitorJob.id}`, txMonitorJob, {
+              delay: blockDelay,
+            });
```

Comment:
Why are we no longer batching the individual page updates?

Reply:
thats all came out of rushed updates

Reply:
Hmm, I see where batching makes it more complicated to track completion later on...

Created tracking issue: #102